scripts/zones/Promyvion-Vahzl/mobs/Provoker.lua

```diff
@@ -0,0 +1,67 @@
+-----------------------------------
+-- Area: Promyvion - Vahzl
+--   NM: Provoker
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/magic")
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:addMod(tpz.mod.DOUBLE_ATTACK, 25)
```

Comment:
considering wiki claims it hits hard as well as having double attack I'm wondering if maybe one of its jobs should be war, but I'll leave well enough alone for now and say this is fine if we have indications besides the additional effect that its a rdm (wiki will often see one thing that looks like what  job has, and then just declare a mob to be that job.. its often wrong or forgets that mobs can have more than 1)

Reply:
There doesn't seem to be any substantial proof that its job is RDM aside from the misconception that it casts its enspell effects. Unless someone more knowledgeable than I am about mob jobs wants to chime in I think I'll go ahead and change its job.

Reply:
https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit#gid=0
I'd go with war/pld with a family bonus of 10% defp per jimmayus/meloetta's data. that would give the double attack.

Reply:
With the job change remember to update the onMobInitialize section. you'd be adding 25 on top of what warrior already gets at this level (setMod instead of addmod would enforce it having only 25, while removing the line would rely on what warrior has native at this lv)